Xamarin.Essentials/Screenshot/Screenshot.shared.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        public static bool IsCaptureAvailable
+            => PlatformCanCapture;
```

Comment:
Not sure what we want to do here...

I see the media picker has a public property, but the app actions does not.